network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/iframe_block.html

```diff
@@ -14,11 +14,15 @@
     </div> <!-- End .cms -->
   </div> <!-- End .col-lg-8 -->
 {% elif page_type == "article" %}
-  {# Break out of the article_pag.html containment grid #}
+  {# Break out of the article_page.html containment grid #}
   </div> <!-- end .offset-sm-1.col-sm-10.col-12.article-blocks -->
+{% else %}
+  {# Mozfest page. Break out of the containment grid #}
+    </div>
+  </div>
 {% endif %}
 
-  <div class="col-lg-12">
+  <div class="{% if self.wide_iframe %}col-lg-12{% else %}col-lg-8 offset-lg-2{% endif %} {% if page_type != 'blog' and page_type != 'article' %}p-0{% endif %}">
```

Comment:
Getting very unmaintable. We had at least three pages to consider:
* Mozfest primary page 
* Blog detail page
* Article page

Reply:
The streamfields-as-containers (rows) work hopefully makes this the last hotfix.